dotCMS/src/main/java/com/dotcms/management/handler/InfoEndpointHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.dotcms.management.handler;
+
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.util.ReleaseInfo;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * CDI-managed endpoint handler that provides basic service information.
+ * This demonstrates how easy it is to add new management endpoints with CDI auto-discovery.
+ */
+@ApplicationScoped
+public class InfoEndpointHandler implements ManagementEndpointHandler {
+
+    /**
+     * Default constructor required for CDI proxy creation.
+     */
+    public InfoEndpointHandler() {
+        // Default constructor for CDI
+    }
+
+    @Override
+    public void handle(HttpServletRequest request, HttpServletResponse response) 
+            throws ServletException, IOException {
+        
+        Logger.debug(this, "Processing info endpoint request");
+        
+        try {
+            // Prepare service information
+            StringBuilder info = new StringBuilder();
```

Comment:
this will be easier to java 21 with heredoc (long interpolable strings)

Reply:
Would love to be able to use Java 21 syntax